activation/post_supervisor.go

```diff
@@ -47,8 +47,11 @@ func DefaultTestPostServiceConfig() PostSupervisorConfig {
 
 type PostSupervisorConfig struct {
 	PostServiceCmd string `mapstructure:"post-opts-post-service"`
+	NodeAddress    string `mapstructure:"post-opts-node-address"`
 
-	NodeAddress string `mapstructure:"post-opts-node-address"`
+	CACert string `mapstructure:"post-opts-ca-cert"`
+	Cert   string `mapstructure:"post-opts-cert"`
+	Key    string `mapstructure:"post-opts-key"`
```

Comment:
Version `v0.5.0-alpha2` does not yet have this CLI parameter, that's why I didn't include it and upgrading to `v0.5.0-alpha3` isn't possible yet because it fails to compile in with `go-spacemeshos/post`.

We can update this section when `v0.5.0` of `post-rs` is released (together with an update of `go-spacemeshos/post`).